airflow/providers/microsoft/azure/provider.yaml

```diff
@@ -85,6 +84,7 @@ dependencies:
   #   See issue https://github.com/apache/airflow/issues/30199
   - azure-mgmt-containerinstance>=7.0.0,<9.0.0
   - azure-mgmt-datafactory>=1.0.0,<2.0
+  - azure-mgmt-containerregistry>=8.0.0
```

Comment:
please place this above line 83
It's not part of the TODO block

Reply:
Sure! Just updated it